views/js/qtiCommonRenderer/renderers/interactions/ChoiceInteraction.js

```diff
@@ -55,13 +55,16 @@ define([
         var $choiceInputs = $container.find('.qti-choice').find('input:radio,input:checkbox').not('[disabled]').not('.disabled');
 
         $choiceInputs.on('keydown.commonRenderer', function(e){
+            var $this = $(this);
             var keyCode = e.keyCode ? e.keyCode : e.charCode;
             if(keyCode !== KEY_CODE_TAB){
                 e.preventDefault();
             }
 
             if( keyCode === KEY_CODE_SPACE || keyCode === KEY_CODE_ENTER){
-                _triggerInput($(this).closest('.qti-choice'));
+                _.delay(function(){
```

Comment:
This PR  is only intended to fix the `space` key selection. The tab navigation has been made inside another PR, related to the test runner only.